test-bench.sh

```diff
@@ -186,11 +186,16 @@ function print_end(){
 
     HTML_OUTPUT_DIR=$(ls -td -- $SDKMS_REST_API_HOME"/target/jmeter/reports/"* | head -n 1)
     CSV_OUTPUT_FILE=$(ls -td -- $SDKMS_REST_API_HOME"/target/jmeter/results/"* | head -n 1)
+    FILE_IDF_SUB="$FILE_IDF"
+    if [ ! -z "$MODE" ]
+    then
+        FILE_IDF_SUB="$MODE"
```

Comment:
@laxmishanagaraj - after this change date won't be present in the file names ?

Reply:
As of today, date is not mentioned even for for FPE/plugin specific tests. Eg for plugin encrypt tests, the filename was "PLUGIN-RSA-1024-50THREADS-120SECONDS-scb-encrypt-AGGREGATE.csv" and for FPE it was "ENCRYPTION-AES-192-50THREADS-120SECONDS-fpe-AGGREGATE.csv".